src/realm/geospatial.hpp

```diff
@@ -117,6 +117,7 @@ struct GeoBox {
 //   - Any interior ring must be entirely contained by the outer ring.
 //   - Interior rings cannot intersect or overlap each other. Interior rings cannot share an edge.
 struct GeoPolygon {
+    GeoPolygon() {}
     GeoPolygon(std::vector<GeoPoint>&& p)
```

Comment:
Ideally these would all be `explicit` conversions. I asked on slack if anyone was relying on them being implicit. Feel free to not handle it in this PR if you'd rather just move on.